src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * AddToCart constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
```

Comment:
Niekoniecznie. Klient podczas zakupu w podanym sklepie wybiera ilość m2 płytek jakie chce kupić i nie jest to integer. To pierwszy lepszy przykład gdyż np. w sklepach z kosmetykami diy może być konieczność określania kupowanej ilości w gramach przy czym cena określana jest dla kg. Wtedy do koszyka dodajesz w praktyce tylko ułamek sztuki produktu.